pages/collection/[id]/index.js

```diff
@@ -66,7 +51,15 @@ export default function Collection({
           collections={collections}
           currentCollection={currentCollection}
         />
-        <StyledLink href={`/collection/${id}/play`}>⏵ play</StyledLink>
+        <Button
+          buttonVariant="play"
+          onClick={(event) => {
+            event.stopPropagation();
+            router.push(`/collection/${id}/play`);
+          }}
+        >
```

Comment:
If you need the element only for redirecting, it should be a link - not a button.